### DISCUSSION TITLE
WikiJS 3 Preview

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#6756] [Help / Questions] [answered]
I saw a comment from @NGPixel that we could try the WikiJS 3 Preview at any time, which I was unaware of.

I got super excited and pulled the latest docker image for v3 and immediately ran into issues with being unable to login.

1. There is no setup / admin creation flow.

2. Register doesn't work out of the box (I thought maybe the first account would be the administrator)

3. The default login credentials were not documented anywhere, and any defaults used in previous versions of WikiJS did not work.

4. After finally editing database manually to make the password admin123, I logged in and went to change the pass to a newly generated password, and I got a bcrypt not found error.

I'm not too familiar with postgres. It took me a bit because the formatting of the wiki.users auth column was formatted weird.

Overall, it was a huge pain the the butt to get going.

At the very least, use default credentials used in other versions of wikijs or document what the new credentials are.

Anyways, I love the work you're doing, and I am very excited for a stable release.

**Comment by P4mes4n** [comment, marked as answer]
Its an alpha, a lot of stuff isnt implemented yet.
The default login worked fine for me, just as documented [here](https://next.js.wiki/docs/install/platform/docker).

**Reply by ghost** [reply]
> Its an alpha, a lot of stuff isnt implemented yet.
> The default login worked fine for me, just as documented [here](https://next.js.wiki/docs/install/platform/docker).

I wasn't aware of documentation for v3. Thanks for the information! This helps a lot.

**Comment by NGPixel** [comment]
The default login is documented in several places:

- https://next.js.wiki/docs/install/platform/docker
- The README of https://github.com/requarks/wiki/tree/vega

The last link has also been mentioned from the [blog post](https://blog.js.wiki/news/2022/wiki-js-3-developer-preview) on this very topic.

A simple google search would have linked you to these resources...

**Reply by ghost** [reply]
@NGPixel

I just took a look and saw docs at docs.requarks.io and js.wiki. I didn't realize js.wiki was a mirror.

I also now see next.js.wiki, I must have missed that last night.

Having 3 separate domains just adds to the confusion. It might be worth consolidating under one domain with a redirect. 


Edit:
Ah, I missed that blog post as well. That was my mistake. I was reading another blog post that you made and found out about the preview in a comment you left.
